github/event_types.go

```diff
@@ -183,7 +185,6 @@ type DeploymentStatusEvent struct {
 	Deployment       *Deployment       `json:"deployment,omitempty"`
 	DeploymentStatus *DeploymentStatus `json:"deployment_status,omitempty"`
 	Repo             *Repository       `json:"repository,omitempty"`
-
```

Comment:
Let's please leave this blank line as a separator for the next section.

Reply:
See how the blank line moved?  Can you please revert this change?

<img width="1728" alt="line-moved-2023-08-15_16-22-43" src="https://github.com/google/go-github/assets/6598971/e8288b48-a700-44f2-aadd-4f04e8741523">